JS/dom.js

```diff
@@ -0,0 +1,170 @@
+
+const header = document.querySelector('header .header-nav');
+let plusOneBtn = document.querySelector('.PlusOne');
+let minusOneBtn = document.querySelector('.MinusOne');
+let quantitySpan = document.getElementById('quantitySpan');
+let smallImages = document.querySelectorAll('.product-small-image');
+let mainImage = document.querySelector('.product-image');
+let mainDiv = document.querySelector('.product-popup');
+let xButton = document.querySelector('.x');
+let priceSpan = document.querySelector('.price');
+let quantityBase = 1;
+let descriptionSpan = document.querySelector('.description');
+let titleSpan = document.querySelector('.product-title');
+let countSpan = document.querySelector('.countSpan')
+const ArrayofCartObjects = JSON.parse(localStorage.getItem('ArrayofCartObjects') || '[]');
+
+// Window Events 
+window.onscroll = () => {
+    if (window.scrollY > 70) header.classList.add('active')
+    else header.classList.remove('active')
+}
+
+window.onload = () => {
+    if (window.scrollY > 70) header.classList.add('active')
```

Comment:
Use searchable and meaningful names for your constants. I can't get what `70` refers too so it would be better to keep it in a variable that has a meaningful name, for example: 
`const HEADER_HEIGHT = 70`